ipfs-cluster/test_tests/old_range.yml

```diff
@@ -0,0 +1,14 @@
+name: Test old Onstep Endstep
+config:
+  nodes: 5
+  selector: app=ipfs-cluster
+  times: 1
+  expected:
+      successes: 0
+      failures: 0
+      timeouts: 0
+steps:
+  - name: Run on old step nodes 2 3 4
+    cmd: echo "old range" > OLD_RANGE
+    on_node: 2
+    end_node: 4
```

Comment:
General comments about the tests:

It seems you rely on information echo-ed to the screen to see that things work and make sense.

Instead you should find a way to verify that the test worked with `assertions`. Maybe the `assertions` part of the DSL does not support verifying that something happened on, say, 1/3 of nodes. Then we probably need to extend that part of the language too. It does not necessarily need to happen in this PR though.